dataload/import_to_elasticsearch.py

```diff
@@ -424,6 +457,50 @@ def grant_generator():
 
         shutil.rmtree(tmp_dir)
 
+    # Clear any query caches
+    cache.clear()
+
+
+# From 360Insights v2
+# bins for organisation age (in days)
+AGE_BINS = [x * 365 for x in [-1, 1, 2, 5, 10, 25, 200]]
+AGE_BIN_LABELS = [
+    "Under 1 year",
+    "1-2 years",
+    "2-5 years",
+    "5-10 years",
+    "10-25 years",
+    "Over 25 years",
+]
+
+
+def to_band(value, bins, labels):
+    bins = bins.copy()
+    if value is None:
+        return None
+    previous_value = bins.pop(0)
+    for i, b in enumerate(bins):
+        if value > previous_value and value <= b:
+            return labels[i]
+        previous_value = b
+
+
+def update_doc_with_first_recipient_org_info(grant):
```

Comment:
Why do this? 
Do we ever get grants with more than one recipient org, or is it not worth thinking about?

Reply:
Elasticsearch doesn't really like indexing on nested data (not sure it's actually possible) and we want a field in the first org info which is the best matching one (from FindThatCharity), it is possible for multiples, such as when an organisation has both a charity and a company. We extract that field to a higher level in the document object so that we can do an aggregate on the field more easily.